File: Python/0010. Vowel.py
# If you need to import additional packages or classes, please import here.

def func():
    # please define the python3 input here. 
    # For example: a,b = map(int, input().strip().split())
    # please finish the function body here.
    # please define the python3 output here. For example: print().
    while True:
        try:
            s = input().strip()
            vowel = ['a','e','i','o','u','A','E','I','O','U']
            for c in s:
                if c!=' ':
                    if c in vowel:
                        c = c.upper()
                    else:
                        c = c.lower()
                print(c,end='')
            print("")
        except EOFError:
            break

    
if __name__ == "__main__":
    func()

File: C++/0010. Vowel.cpp
// we have defined the necessary header files here for this problem.
// If additional header files are needed in your program, please import here.
#include <vector>
#include <algorithm>
int main()
{  
  // please define the C++ input here. For example: int a,b; cin>>a>>b;;  
  // please finish the function body here.  
  // please define the C++ output here. For example:cout<<____<<endl; 
    string diary;
    while (getline(cin,diary)){
        vector<char> vowel{'a','e','i','o','u','A','E','I','O','U'};
        for(auto s:diary){
            if(s!=' '){
                if(std::find(vowel.begin(),vowel.end(),s)!=vowel.end()){
                    s = toupper(s);
                }
                else{
                    s = tolower(s);
                }
            }
            cout<<s;
        }
        cout<<endl;
    }
    return 0;
}
